src/ml_tooling/utils.py

```diff
@@ -362,3 +362,30 @@ def _find_src_dir(path: pathlib.Path = None, max_level: int = 2) -> pathlib.Path
     raise MLToolingError(
         f"No modules found in {output_folder}! Is there an __init__.py file in your module?"
     )
+
+
+def _classify(x: pd.DataFrame, estimator: Estimator, threshold: float = 0.5):
+    """
+    Make a binary classification of prediction probabilities with the given threshold
+
+    Parameters
+    ----------
+    x: pd.DataFrame
+        The data to use for classification
+    estimator: Estimator
+        The estimator to use for making the prediction
+    threshold: float
+        Threshold of classification
+
+    Returns
+    -------
+    np.array
+        Array of class predictions
+    """
+    y_prob = estimator.predict_proba(x)
```

Comment:
If we have a check for the binary classification (something like `len(estimator.classes_) == 2`) we can do a simple comparison: `return (y_pred[:, 1] > threshold).astype(np.int32)`

In binary classification 0 will always be negative class and 1 positive class

Reply:
Yeah, this is way simpler :)